modules/ROOT/pages/salesforce/salesforce-connector-100.adoc

```diff
@@ -150,7 +151,7 @@ https://www.mulesoft.com/exchange/?search=salesforce&type=template[Anypoint Exch
 
 * Account Broadcast template
 +
-https://www.mulesoft.com/exchange/org.mule.templates/template-sfdc2sfdc.wday.sap.db-account-broadcast/[Salesforce to Salesforce, Workday, SAP, and Database]
+https://www.mulesoft.com/exchange/org.mule.templates/template-sfdc2sfdc.wday.sap.db-account-broadcast/[Salesforce to Salesforce, Workday, SAP, and database account broadcast]
```

Comment:
Does this mean a Salesforce broadcast to each of Salesforce, Workday, SAP, and a database account (was this supposed to be Database Connector? Just checking...)?
Or does this mean Salesforce-to-Salesforce, Workday, SAP, and database account broadcasts?
I guess all I am asking is that you verify the word order to make sure that adjectives and nouns are distinct.